channel.go

```diff
@@ -189,6 +189,21 @@ func (ch *Channel) AddModerators(userIDs ...string) error {
 	return ch.client.makeRequest(http.MethodPost, p, nil, data, nil)
 }
 
+// InviteMembers invites users with given IDs to the channel
+func (ch *Channel) InviteMembers(userIDs ...string) error {
+	if len(userIDs) == 0 {
+		return errors.New("user IDs are empty")
```

Comment:
`return nil`

Reply:
im following the same convention used here https://github.com/GetStream/stream-chat-go/pull/24/files#diff-861e77702f49a71d32be4b3c7600818fR210

Reply:
We define conventions now? 😆